Havtorn/Source/Editor/EditorManager.cpp

```diff
@@ -46,7 +46,7 @@ namespace Havtorn
 		SAFE_DELETE(ResourceManager);
 	}
 
-	bool CEditorManager::Init(const CGraphicsFramework* framework, const CWindowHandler* windowHandler, CRenderManager* renderManager, CSequencerSystem* sequencerSystem)
+	bool CEditorManager::Init(const CGraphicsFramework* framework, const CWindowHandler* windowHandler, CRenderManager* renderManager, CSequencerSystem* /*sequencerSystem*/)
```

Comment:
Can we remove the CSequencerSystem* parameter here? (Might come back if I see other changes that explain why we want to keep this)

```suggestion
	bool CEditorManager::Init(const CGraphicsFramework* framework, const CWindowHandler* windowHandler, CRenderManager* renderManager)
```